### PR TITLE
VaultId from Graph, new addresses and subgraph url

### PR DESCRIPTION
**Pull request by devey**
- Retrieve correct `VaultId` if user has one with selected `shortOToken`.
- Use new Exchange and Handler addresses that have updated events for `OptionsBough` and `OptionsSold`.
- Update Subgraph URL to the new Studio deployment, as recommended by The Graph.

### CODE REVIEW

File: packages/front-end/src/hooks/useSellOperate/index.tsx [hunk 1]
```diff
@@ -111,7 +132,7 @@ const useSellOperate = (): [
               owner: addressOrDefault,
               secondAddress: addressOrDefault,
               asset: ZERO_ADDRESS,
-              vaultId: nextVaultId, // TODO vaultId, each different short position the user holds will be held in a unique vaultId, for now putting it in new vaults
+              vaultId: vaultId, // TODO vaultId, each different short position the user holds will be held in a unique vaultId, for now putting it in new vaults
```
Comment (MeanBoyCousin) [inline packages/front-end/src/hooks/useSellOperate/index.tsx:135]: Is this TODO comment still needed? Just wondered as you seem to have changed the variable name to what was mentioned in the original comment.

**Review by MeanBoyCousin — CHANGES_REQUESTED** [review]
Will need to update the `REACT_APP_SUBGRAPH_URL` env variable on Vercel for the testnet app and for previews on the main app.

File: packages/front-end/src/hooks/useSellOperate/index.tsx [hunk 2]
```diff
@@ -66,19 +66,40 @@ const useSellOperate = (): [
...
     `,
     {
-      variables: { account: address?.toLowerCase() },
+      variables: {
+        account: address?.toLowerCase(),
+        oToken: oToken?.toLowerCase(),
+      },
     }
   );
+  let vaultId = BigNumber.from(0);
```
[the remark this answers is not in the document]
Comment (devey) [reply packages/front-end/src/hooks/useSellOperate/index.tsx:94]: done

File: packages/front-end/src/hooks/useSellOperate/index.tsx (hunk 1 above)
Comment (devey) [reply packages/front-end/src/hooks/useSellOperate/index.tsx:135]: right

**Pull request merged**